src/main/resources/com/mathworks/ci/tools/MatlabInstaller/help-version.html

```diff
@@ -0,0 +1,14 @@
+<div>
+  <p>
+    Enter the MATLAB version to be installed. For update versions, append the appropriate update number to the MATLAB version.
```

Comment:
Insert the MATLAB version to install. To install the latest release of MATLAB, insert `latest` in the **Version** box.
- To install the latest update of a release, specify only the release name, for example, `R2023b`.
- To install a specific update release, specify the release name with an update number suffix, for example, `R2023bU4`.
- To install a release without updates, specify the release name with an update 0 or general release suffix, for example, `R2023bU0` or `R2023bGR`.

**Example:**  `R2023b`
**Example:**  `latest`
**Example:**  `R2023bU4`


**Note:** The plugin does not install dependencies on a Linux platform. If you are using a Linux platform, verify that the required software is available before installing products using MATLAB Package Manager. For more information, see <a href="https://in.mathworks.com/help/install/ug/get-mpm-os-command-line.html">Get MATLAB Package Manager</a>.

Reply:
(Again, consider removing the link from the help text.)

Reply:
Done. keeping link here with same rationale that it would be quicker for user to access the information and close to the plugin.